joint_trajectory_controller/package.xml

```diff
@@ -16,41 +18,24 @@
   <buildtool_depend>catkin</buildtool_depend>
 
   <build_depend>cmake_modules</build_depend>
-  <build_depend>actionlib</build_depend>
-  <build_depend>angles</build_depend>
-  <build_depend>roscpp</build_depend>
-  <build_depend>urdf</build_depend>
 
-  <build_depend>control_toolbox</build_depend>
-  <build_depend>controller_interface</build_depend>
-  <build_depend>hardware_interface</build_depend>
-  <build_depend>realtime_tools</build_depend>
+  <depend>actionlib</depend>
+  <depend>angles</depend>
+  <depend>roscpp</depend>
+  <depend>urdf</depend>
 
```

Comment:
remove empty lines between `<depend>` elements, if any (from the diff it seems there's this one and another one below)